### ISSUE TITLE
Non existing window

### ISSUE THREAD

**Issue by drdeimos** [baskerville/bspwm#604]
Sometimes i'm watching  nonexisting window.
Tiling area occupied, as it were transparent window.
Example:

```
drdeimos@UbuSSD ~ $ bspc query -T -m ^1|jq '.desktops[2].root.secondChild'
{
  "id": 44040416,
  "splitType": "vertical",
  "splitRatio": 0.5,
  "birthRotation": 0,
  "vacant": false,
  "hidden": false,
  "sticky": false,
  "private": false,
  "locked": false,
  "presel": null,
  "rectangle": {
    "x": 683,
    "y": 18,
    "width": 683,
    "height": 750
  },
  "firstChild": null,
  "secondChild": null,
  "client": {
    "className": "Chromium-browser",
    "instanceName": "chromium-browser",
    "borderWidth": 2,
    "state": "tiled",
    "lastState": "tiled",
    "layer": "normal",
    "lastLayer": "normal",
    "urgent": false,
    "shown": true,
    "tiledRectangle": {
      "x": 683,
      "y": 18,
      "width": 679,
      "height": 746
    },
    "floatingRectangle": {
      "x": 2,
      "y": 52,
      "width": 1362,
      "height": 746
    }
  }
}
```

but chromium not running!

```
drdeimos@UbuSSD ~ $ ps -eF |grep chromi
drdeimos 21998 20970  0  3556  1088   2 08:36 pts/1    00:00:00 grep chromi
drdeimos@UbuSSD ~ $ 
```

![screenshot1](https://cloud.githubusercontent.com/assets/3472810/21795164/4d54ba96-d711-11e6-9a5b-5d41bbf1e131.png)

**Comment by drdeimos** [comment]
[get_status.txt](https://github.com/baskerville/bspwm/files/695352/get_status.txt)
[dumpstate.txt](https://github.com/baskerville/bspwm/files/695353/dumpstate.txt)

**Comment by msteen** [comment]
I wanted to test whether this could be caused by `bspc wm --load-state`, so I planned to open a terminal next to my already open window, to write the output of `bspc wm --dump-state` to some file in my home folder, to reload `bspwm`, to kill the terminal window, and to then call `bspc wm --load-state` with the state file to see whether that caused it. However when I changed my desktop layout to tiled, I happened to already have reproduced the issue, which I never saw happen before, so it is quite a coincidence. Anyway, I tried to reproduce as mentioned above and I managed to do so, this means that `bspc wm --load-state` is likely to be the cause of this problem and needs to be fixed by trimming nodes which no longer have a window or something similar.
